src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -81,9 +97,29 @@ const AnnotationActivity = ({
     const activityLinkMessage = isFileVersionUnavailable
         ? messages.annotationActivityVersionUnavailable
         : { ...linkMessage, values: { number: linkValue } };
+    const clickHandlers = useSuppressiveClicks<HTMLDivElement>({ onClick: handleOnSelect });
+    const interactiveDivProps = isInteractive
+        ? {
```

Comment:
Should we leave these props on but add `aria-disabled`, instead? Otherwise, we're working around some eslint rules regarding event handlers on non-interactive elements.